cartoframes/viz/legend.py

```diff
@@ -15,13 +15,27 @@ class Legend(object):
         title (str, optional): Title of legend.
         description (str, optional): Description in legend.
         footer (str, optional): Footer of legend. This is often used to attribute data sources.
+        variable (optional): If the information in the legend depends on a different value than the
```

Comment:
What type is `variable`? `str` or something else?

Reply:
ouch! str, I'm gonna fix this